examples/nlp/md/semantic_similarity_with_bert.md

```diff
@@ -322,7 +322,7 @@ with strategy.scope():
     # Freeze the BERT model to reuse the pretrained features without modifying them.
     bert_model.trainable = False
 
-    bert_output = bert_model(
+    bert_output = bert_model.bert(
```

Comment:
Why this change?

Reply:
Sorry, I forgot to clarify this change. When I tried to save and load the trained model I got this error `ValueError: The two structures don't have the same nested structure.`
I changed this line following the solution for this issue in the HugingFace repo https://github.com/huggingface/transformers/issues/3627